slider-rfc.md

```diff
@@ -0,0 +1,122 @@
+Slider API RFC
+
+### Goal
+
+Create an accessible, easy to use but still flexible enough slider component.
+
+Following our renderlesskit slider component:
+[renderlesskit/react slider](https://github.com/timelessco/renderlesskit-react/blob/master/docs/slider.md)
+
+### Components
+
+- Slider - Main component
+- Optional (only for flexiblity)
```

Comment:
I think you remove this RFC from Source Code.
We have an Issue Reference for this already.